### DISCUSSION TITLE
[Feature Request]: 名称识别相关

### DISCUSSION THREAD

**Discussion by 99Mr-Z** [jxxghp/MoviePilot#3443] [讨论]
### 当前程序版本

1.61

### 运行环境

Docker

### 功能改进类型

主程序

### 功能改进

1，能否将未识别名称未识别部分标记出来，方便用户手动进行矫正补充词表
例如：[Airota&VCB-Studio] Kiniro Mosaic Pretty Days [Ma10p_1080p][x265_flac_aac]
这个名称无法识别，能够在返回结果中给出标记，或者给出识别转换的结果来验证手动添加的词表是否生效了，这样更加方便添加手动添加词表。
2，能否针对一个已有的命名规范进行转换，例如VCB压制组的命名规范（参考附图）通过简单的方式转换成正则表达式或者其他方式进行快速识别
3，或者为目录同步增添手动的方式，手动对将要扫描的文件夹逐个进行校对，提前进行标记剧集名称或者系列名称进行手动纠正。

以上问题主要是大批量的本地源文件文件需要一次性使用movie pilot重新进行入库整理
![屏幕截图 2024-01-31 160645](https://github.com/jxxghp/MoviePilot/assets/117721493/db1b8509-c0ae-4520-86d4-126de9341e64)

**Comment by 99Mr-Z** [comment]
命名规范参考链接：
https://github.com/vcb-s/VCB-S_Collation/blob/master/%E8%A7%86%E9%A2%91%E5%92%8C%E5%AD%97%E5%B9%95%E6%95%B4%E7%90%86%E8%A7%84%E8%8C%83.md

**Comment by EkkoG** [comment]
图裂了，粘贴进去后等输入框中的文案变成一个链接后再提交，提交前可以预览一下，是看得出来的

**Comment by 99Mr-Z** [comment]
> 图裂了，粘贴进去后等输入框中的文案变成一个链接后再提交，提交前可以预览一下，是看得出来的

好的，第一次用GitHub不怎么熟悉，谢谢提醒，已经编辑修正了

**Comment by Mister-album** [comment]
支持，moviepilot几乎没有搜索到vcb的资源过，我都是手动添加后加上moviepilot的标签来刮削的

**Comment by 99Mr-Z** [comment]
> 支持，moviepilot几乎没有搜索到vcb的资源过，我都是手动添加后加上moviepilot的标签来刮削的

我现在时手动映射进行预处理文件夹，自动化处理太容易出错了
